### PR TITLE
Support syscoin

### PR DESCRIPTION
**Pull request by tomatoskittles**
We would like to add Syscoin and Syscoin Testnet. Please let me know if there is anything else you would like me to do.

### CODE REVIEW

File: _data/chains/eip155-57.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Syscoin Mainnet",
+  "chain": "SYS",
+  "network": "mainnet",
+  "rpc": [
+    "https://nevm.syscoin.org/api/eth-rpc",
+    "https://web3.syscoin.org",
+    "wss://web3.syscoin.org"
+  ],
+  "faucets": ["https://faucet.syscoin.org"]
```
Comment (ligi) [inline _data/chains/eip155-57.json:10]:
```suggestion
  "faucets": ["https://faucet.syscoin.org"],
```

File: _data/chains/eip155-5700.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Syscoin Tanenbaum Testnet",
+  "chain": "SYS",
+  "network": "testnet",
+  "rpc": [
+    "https://tanenbaum.io/api/eth-rpc"
+  ],
+  "faucets": ["https://faucet.tanenbaum.io"]
```
Comment (ligi) [inline _data/chains/eip155-5700.json:8]:
```suggestion
  "faucets": ["https://faucet.tanenbaum.io"],
```

**Review by ligi — CHANGES_REQUESTED** [review]
there are 2 commas missing - also the explorers need the `standard` field

**Comment by tomatoskittles** [comment]
> there are 2 commas missing - also the explorers need the `standard` field

Thanks for the review.

I have fixed the comma related syntax error, and I have also removed the explorers. We are using Blockscout, which I don't believe is EIP3091 compliant. xDai also uses Blockscout, and their entry doesn't have explorers either, so I figured this was a good approach.

Hopefully it is good, now.

**Review by ligi — APPROVED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `sys` more than once

**Comment by tomatoskittles** [comment]
> short name must be unique - but found `sys` more than once

I've changed the testnet shortname, to honour the uniqueness requirement.

**Pull request merged**